app/scripts/services/resourceAlerts.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("ResourceAlertsService",
+           function($filter,
+                    AlertMessageService,
+                    Navigate,
+                    QuotaService) {
+    var annotation = $filter('annotation');
+    var deploymentStatus = $filter('deploymentStatus');
+    var getGroupedPodWarnings = $filter('groupedPodWarnings');
+
+    var alertHiddenKey = function(alertID, namespace) {
+      return 'hide/alert/' + namespace + '/' + alertID;
+    };
+
+    var isAlertHidden = function(alertID, namespace) {
+      var key = alertHiddenKey(alertID, namespace);
+      return localStorage.getItem(key) === 'true';
+    };
+
+    var hideAlert = function(alertID, namespace) {
+      var key = alertHiddenKey(alertID, namespace);
+      localStorage.setItem(key, 'true');
+    };
```

Comment:
isnt this duplicating these that we already had put into AlertMessageService, which is actually referenced in the quota warnings below.